### ISSUE TITLE
Invalid dependencies or related test packages during Unity Cloud Build

### ISSUE THREAD

**Issue by elainesong7** [jelte/be.khepri.play.assetdelivery.addressables#39]
Hi, I am facing some invalid independencies problem when I do the unity cloud build, but no issues for local build.

Error messages : 
[Unity]   Package be.khepri.play.assetdelivery.addressables@https://github.com/jelte/be.khepri.play.assetdelivery.addressables.git#1.0.0-rc5 has invalid dependencies or related test packages:
79: [Unity]     com.google.play.assetdelivery (dependency): Package [com.google.play.assetdelivery@1.2.0] cannot be found
80: [Unity]   Package com.google.play.assetdelivery@1.2.0 has invalid dependencies or related test packages:
81: [Unity]     com.google.play.common (dependency): Package [com.google.play.common@1.2.0] cannot be found
82: [Unity]     com.google.play.core (dependency): Package [com.google.play.core@1.2.0] cannot be found
83: [Unity]   Package com.google.play.common@1.2.0 has invalid dependencies or related test packages:
84: [Unity]     com.google.android.appbundle (dependency): Package [com.google.android.appbundle@1.2.0] cannot be found
85: [Unity]   Package com.google.play.core@1.2.0 has invalid dependencies or related test packages:
86: [Unity]     com.google.android.appbundle (dependency): Package [com.google.android.appbundle@1.2.0] cannot be found
87: [Unity] A re-import of the project may be required to fix the issue or a manual modification of /BUILD_PATH/xxxx/Packages/manifest.json file.

Before I import your package, I already had the GooglePlayPlugin in my project and the version is 1.4.0, is it possible to use 1.4.0 version ?

**Comment by elainesong7** [comment]
> Yes, I've been using 1.4.0 for a while and now using 1.5.0 Google Play Plugin and it is working fine.
> I just removed the dependencies from package.json.

Hi, I removed all the dependency from package.json, but still have cloud build error with invalid reference
_com.google.play.assetdelivery (dependency): Package [com.google.play.assetdelivery@1.2.0] cannot be found_

Is my package.json correct ?? Should I change the version to 1.0.0-rc5 ? 
{
  "name": "be.khepri.play.assetdelivery.addressables",
  "version": "1.0.0-rc5",
  "displayName": "Google Play Asset Delivery - Addressables",
  "description": "Provide addressables with support for Play Asset Delivery",
  "unity": "2018.6",
  "author": {
    "name": "Jelte Steijaert",
    "e-mail": "jelte.steijaert@gmail.com"
  },
  "publishConfig": {
    "registry": "https://npm.pkg.github.com/@jelte"
  },
  "type": "library",
  "hideInEditor": false
}

**Comment by a-l-e-x-d-s-9** [comment]
That's how my package.json looks like: 
```json
{
  "name": "be.khepri.play.assetdelivery.addressables",
  "version": "1.0.0-rc4",
  "displayName": "Google Play Asset Delivery - Addressables",
  "description": "Provide addressables with support for Play Asset Delivery",
  "unity": "2018.6",
  "dependencies": {
  },
  "author": {
    "name": "Jelte Steijaert",
    "e-mail": "jelte.steijaert@gmail.com"
  },
  "publishConfig": {
    "registry": "https://npm.pkg.github.com/@jelte"
  },
  "type": "library",
  "hideInEditor": false
}
```
I don't think that the string in the version field has any influence on anything.
I'm using local build and not cloud build, so not sure exactly what may be going wrong with that, but if I had to guess, you probably haven't updated the file in the right directory, or it's cached somewhere so it's using the wrong version.

**Comment by elainesong7** [comment]
> That's how my package.json looks like:
> 
> ```json
> {
>   "name": "be.khepri.play.assetdelivery.addressables",
>   "version": "1.0.0-rc4",
>   "displayName": "Google Play Asset Delivery - Addressables",
>   "description": "Provide addressables with support for Play Asset Delivery",
>   "unity": "2018.6",
>   "dependencies": {
>   },
>   "author": {
>     "name": "Jelte Steijaert",
>     "e-mail": "jelte.steijaert@gmail.com"
>   },
>   "publishConfig": {
>     "registry": "https://npm.pkg.github.com/@jelte"
>   },
>   "type": "library",
>   "hideInEditor": false
> }
> ```
> 
> I don't think that the string in the version field has any influence on anything.
> I'm using local build and not cloud build, so not sure exactly what may be going wrong with that, but if I had to guess, you probably haven't updated the file in the right directory, or it's cached somewhere so it's using the wrong version.

Thanks for reply, I will check on it. 
By the way may I know what is your Google Play Packages path ? Are you install to your project's Assets directory or install to your project's Packages directory ?

**Comment by taictdev** [comment]
I fixed this bug
- In packages-lock.json (at folder Packages) and package.json (at Library\PackageCache\be.khepri.play.assetdelivery.addressables), remove all item in "dependencies" bellow:
              "dependencies": {
                  "be.khepri.play.assetdelivery.addressables": {
                    "version": "https://github.com/jelte/be.khepri.play.assetdelivery.addressables.git#1.0.0-rc4",
                    "depth": 0,
                    "source": "git",
                    **"dependencies": {},**
                  },
-

**Comment by jelte** [comment]
Google removed the entire UPM registry, you now need to install the Google Play AssetDelivery via the UnityPackage from https://github.com/google/play-unity-plugins/releases .
This is fixed in version 1.0.0
